specs/commandline/blob.md

```diff
@@ -0,0 +1,504 @@
+```
+
+The trust policy configuration in the JSON file should be validated according to [trust policy properties](https://github.com/notaryproject/notaryproject/specs/trust-store-trust-policy.md#trust-policy-properties). A successful message should be printed out if trust policy configuration are imported successfully. Error logs including the reason should be printed out if the importing fails.
+
+If there is an existing trust policy configuration, prompt for users to confirm whether discarding existing configuration or not. Users can use `--force` flag to discard existing trust policy configuration without prompt.
+
+### Show blob trust policies
+
+Use the following command to show trust policy configuration:
+
+```shell
+notation blob policy show
+```
+
+Upon successful execution, the trust policy configuration is printed out to standard output. If trust policy is not configured or is malformed, users should receive an error message via standard error output, and a tip to import trust policy configuration from a JSON file.
+
+### Export blob trust policy configuration into a JSON file
+
+Users can redirect the output of command `notation blob policy show` to a JSON file.
+
+```shell
+notation blob policy show > ./blob_trust_policy.json
+```
+
+### Update trust policy configuration
+
+The steps to update blob trust policy configuration:
+
+1. Export trust policy configuration into a JSON file.
+
+   ```shell
+   notation blob policy show > ./blob_trust_policy.json
+   ```
+
+2. Edit the exported JSON file "blob_trust_policy.json", update trust policy configuration and save the file.
+3. Import trust policy configuration from the file.
+
+   ```shell
+   notation blob policy import ./blob_trust_policy.json
+   ```
+
+## Verify blob signatures
+The `notation blob verify` command can be used to verify blob signatures. In order to verify signatures, user will need to setup a trust policy file `trustpolicy.blob.json` with Policies for blobs. Below are two examples of how a policy configuration file can be setup for verifying blob signatures.
+
+- The Policy named "wabbit-networks-policy" is for verifying blob artifacts signed by Wabbit Networks.
+- Policy "wildcard-verification-policy" is for auditing verification results when user wants to apply a wildcard policy by not providing `--policy-name` argument in `notation blob verify` command.
+
+```jsonc
+{
+    "version": "1.0",
+    "policyType": "blob",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-policy",
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": [ 
+              "ca:wabbit-networks",
+            ],
+            "trustedIdentities": [
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Security Tools"
+            ]
+        },
+        {
+            "name": "wildcard-verification-policy",
+            "wildcardPolicy": true,
```

Comment:
Will there only be one wildcard policy?

Reply:
Correct. Just like OCI policies, there can only be one wildcard policy for blob policies